extensions-contrib/scan-query/src/main/java/io/druid/query/scan/EventHolder.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Druid - a distributed column store.
```

Comment:
this is the old header, please change it to the new ones like you'll find in other files